content/es/docs/languages/go/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Go
+description: >-
+  <img width="35" class="img-initial" src="/img/logos/32x32/Golang_SDK.svg"
+  alt="Go"> Una implementación específica del lenguaje OpenTelemetry en Go.
+aliases: [/golang, /golang/metrics, /golang/tracing]
```

Comment:
The aliases predate this `es` version of the page, so the aliases aren't needed:

```suggestion
# aliases: [/golang, /golang/metrics, /golang/tracing] - aliases not needed since they predate the creation of this page
```